### PR TITLE
Improve sleep documentation; callout function runs list limitation

### PR DESCRIPTION
**Pull request by cdzombak**

### CODE REVIEW

**Comment by cdzombak** [comment]
preview: https://website-76ndf5nuj-inngest.vercel.app/docs/features/inngest-functions/steps-workflows/sleeps

**Review by charlypoly — APPROVED** [review]

File: pages/docs/features/inngest-functions/steps-workflows/sleeps.mdx [hunk 1]
```diff
@@ -3,17 +3,15 @@ import { Callout, GuideSelector, GuideSection, CodeGroup } from "src/shared/Docs
 
 # Sleeps
 
-Two step methods are available to pause the execution of your function for a specific amount of time.
+Two step methods, `step.sleep` and `step.sleepUntil`, are available to pause the execution of your function for a specific amount of time. Your function can sleep for seconds, minutes, or days, up to a maximum of one year (seven days for account on our [free tier](/pricing)).
```
Comment (djfarrelly) [inline pages/docs/features/inngest-functions/steps-workflows/sleeps.mdx:6]:
```suggestion
Two step methods, `step.sleep` and `step.sleepUntil`, are available to pause the execution of your function for a specific amount of time. Your function can sleep for seconds, minutes, or days, up to a maximum of one year (seven days for account on our [free tier](/pricing?ref=docs-sleeps)).
```

File: pages/docs/features/inngest-functions/steps-workflows/sleeps.mdx [hunk 2]
```diff
@@ -163,17 +161,15 @@ Check out the [`step.sleep_until()` Python reference.](/docs/reference/python/st
...
-
-
-
 <Callout variant={'info'}>
-**About: Maximum sleep duration**
+  **Sleeps and trace/log history**
 
-Functions can sleep for a year (seven days for the [free tier plans](/pricing)).
-</Callout>
+  You may notice that Inngest Cloud's Function Runs view doesn't show function runs that use sleeps longer than your [Inngest plan's](/pricing) trace & log history limit, even though the functions are still sleeping and will continue to run as expected. **This is a known limitation** in our current infrastructure and we're working to improve it.
```
Comment (djfarrelly) [inline pages/docs/features/inngest-functions/steps-workflows/sleeps.mdx:169]: **suggestions**:
* Add ref tag
* Clarify that it's not an infra limitation, it's a product limitation. We don't want to give the idea that our infrastructure cannot handle this, it's a UI/UX issue.

```suggestion
  You may notice that Inngest Cloud's Function Runs view doesn't show function runs that use sleeps longer than your [Inngest plan's](/pricing?ref=docs-sleeps) trace & log history limit, even though the functions are still sleeping and will continue to run as expected. **This is a known limitation** in our current dashboard and we're working to improve it.
```

**Review by djfarrelly — COMMENTED** [review]
Suggested adjustments - thanks Chris for improving these docs1

**Pull request merged**